README.md

```diff
@@ -231,6 +231,15 @@ Then in your `kuzzlerc` file, you can change the following configuration variabl
   }
 }
 ```
+If needed, you can override `bucketName` by adding `bucketName` arg to the request:
```

Comment:
You need to update the doc/2 documentation accordingly with the new parameters

Reply:
It is done